src/analysis/normed_space/basic.lean

```diff
@@ -334,6 +346,18 @@ have tendsto f a (𝓝 0) ↔ tendsto (λ e, ∥ f e - 0 ∥) a (𝓝 0) :=
   tendsto_iff_norm_tendsto_zero,
 by simpa
 
+lemma squeeze_zero_norm' {f : γ → α} {g : γ → ℝ} {t₀ : filter γ}
+  (h : ∀ᶠ n in t₀, ∥f n∥ ≤ g n)
+  (h' : tendsto g t₀ (𝓝 0)) : tendsto f t₀ (𝓝 0) :=
+tendsto_zero_iff_norm_tendsto_zero.mpr
+  (squeeze_zero' (t₀.eventually_of_forall (λ n, norm_nonneg _)) h h')
+
+lemma squeeze_zero_norm {f : γ → α} {g : γ → ℝ} {t₀ : filter γ}
+  (h : ∀ (n:γ), ∥f n∥ ≤ g n)
+  (h' : tendsto g t₀ (𝓝 0)) :
+  tendsto f t₀ (𝓝 0) :=
+tendsto_zero_iff_norm_tendsto_zero.mpr (squeeze_zero (λ n, norm_nonneg _) h h')
```

Comment:
I'd reuse `squeeze_zero_norm'` here to show how these two lemmas are related. And it might be a good idea to add docstrings mentioning prime/non-prime versions.

```suggestion
squeeze_zero_norm' (eventually_of_forall _ h) h'
```